recipes/blast/meta.yaml

```diff
@@ -6,11 +6,20 @@ package:
   version: {{ version }}
 
 source:
-  url: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-src.tar.gz
-  sha256: {{ sha256 }}
+  - url: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-src.tar.gz
+    sha256: {{ sha256 }}
+    folder: blast
+  # Get optional RpsbProc
+  # The rpsbproc command line utility is an addition to the standalone version of
+  # Reverse Position-Specific BLAST (RPS-BLAST), also known as CD-Search (Conserved
+  # Domain Search).
+  - url: https://ftp.ncbi.nih.gov/pub/mmdb/cdd/rpsbproc/RpsbProc-src.tar.gz
+    sha256: 8b29f7e7d0a0654d10943e362557ed2d8df9a16218a4c8441c4dea4e7dc9154f
+    folder: RpsbProc
 
 build:
-  number: 4
+  number: 5
+  skip: True # [linux]
```

Comment:
I support this is just for testing this PR?

Reply:
yes it was just to test on OSX as it takes hours to pass the linux test